### PR TITLE
Renderer configuration

### PR DESCRIPTION
**Pull request by cristianvasquez**
This pull request adds the posiblitiy of configuring the following parameters through `trifid-config`

```js
  compactMode: true,
  embedBlanks: true,
  technicalCues: true,
  preferredLanguages: ['en', 'fr', 'de', 'it'],
  highLightLanguage: 'en',
  embedNamed: false,
  embedLists: true,
  debug: false,
  maxLevel: 3
```
Additionally, each parameter can be overwritten through a parameter in the query-request.

The pull request includes example trifid instances

### CODE REVIEW

File: examples/config/adams.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+middlewares:
+
+  welcome:
+    module: trifid-core/middlewares/view.js
+    paths: /
+    methods: GET
+    config:
+      path: examples/static/welcome.hbs
+
+#  entity-renderer:
```
Comment (ludovicm67) [inline examples/config/adams.yaml]: @cristianvasquez can this comment be removed?

File: examples/data/adams.ttl [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  ex:livesWith  ex:morticia ;
+  ex:hasPartner ex:morticia ;
+  ex:hasBrother ex:uncle ;
+  ex:hasChild   ex:pugsley ;
+  ex:hasChild   ex:wednesday ;
+  rdfs:label    "Gomez Addams" ;
+  ex:lives_at   [ rdfs:label "Address" ;
+                  ex:street  "Cemetery Lane" ;
+                  ex:number  "001" ;
+#                   ex:_namedNode [ ex:property "value" ; ex:property "value"]
```
Comment (ludovicm67) [inline examples/data/adams.ttl]: Same here, can this comment beremoved?

**Review by ludovicm67 — APPROVED** [review]
Looks good 👍
Maybe just check if the commented parts are still relevant to keep, if not, just remove them.

File: examples/config/adams.yaml (hunk 1 above)
Comment (cristianvasquez) [reply examples/config/adams.yaml]: Of course!

**Comment by cristianvasquez** [comment]
Thanks for the review @ludovicm67 !

**Review by ludovicm67 — APPROVED** [review]
LGTM

**Pull request merged**